content/applications/hr/appraisals/skills_evolution.rst

```diff
@@ -0,0 +1,149 @@
+   :alt: A report showing all the skills grouped by employee.
+
+.. _appraisals/identify-skills:
+
+Use case: Identify employees with specific skills
+=================================================
+
+Since the :guilabel:`Appraisal Skills Report` organizes all skills by employee, it can be difficult
+to find employees with a specific skill at a specific level. To find these employees, a custom
+filter must be used.
+
+In this example, this report is modified to show employees with an expert level of Javascript
+knowledge. To view only those employees, first remove all active filters in the search bar.
+
+Next, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the search bar, then click
+:guilabel:`Add Custom Filter` beneath the :icon:`fa-filters` :guilabel:`Filters` column to load an
+:guilabel:`Add Custom Filter` pop-up window.
+
+Using the drop-down menu, select :guilabel:`Skill` for the first drop-down, then select
+:guilabel:`Javascript` for the third drop-down field.
+
+Next, click the :guilabel:`New Rule` button, and another line appears. In this second line, select
+:guilabel:`Current Skill Level` for the first drop-down, then select :guilabel:`Expert` for the
+third drop-down field.
+
+After the :guilabel:`New Rule` button is clicked, the word :guilabel:`"any"` in the sentence
+:guilabel:`Match any of the following rules:` changes from plain text into a drop-down menu. Click
+the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon after the word :guilabel:`any`, and select
+:guilabel:`all`.
+
+Finally, click the :guilabel:`Add` button.
+
+.. image:: skills_evolution/javascript.png
+   :align: center
+   :alt: The Custom Filter pop-up with the parameters set.
+
+Now, only employees that have an :guilabel:`Expert` level for the skill :guilabel:`Javascript`
+appear. In this example, only :guilabel:`Mark Demo` meets these criteria.
+
+.. image:: skills_evolution/results.png
+   :align: center
+   :alt: The employees with expert Javascript skills.
+
+Use case: Assess highest improvement
+====================================
+
+Another way to modify the :guilabel:`Appraisal Skills Report` is to identify the employee who has
+the highest amount of improved skills over a specific period of time. In this example, the employee
+with the most amount of improved skills for the third quarter is being determined.
+
+To view this information, first remove the default filter in the search bar. Next, click the
+:icon:`fa-caret-down` :guilabel:`(down arrow)` icon in the search bar, then click
+:guilabel:`Improvement` beneath the :icon:`fa-filter` :guilabel:`Filters` column. Enabling this
+filter only presents skills that have improved.
+
+With the search bar still expanded, click :guilabel:`Add Custom Filter` at the bottom of the
+:icon:`fa-filter` :guilabel:`Filters` column, and an :guilabel:`Add Custom Filter` pop-up window
+appears.
+
+Select :guilabel:`Create Date` for the first drop-down field, then select :guilabel:`is between` for
```

Comment:
During my run-through, I am unable to locate a 'Create Date' option for the first drop-down field...

![Screenshot 2024-09-18 103043](https://github.com/user-attachments/assets/1187c815-f06a-481e-ae94-1a5e8be71afe)